lib/ansible/modules/system/lvol_thin.py

```diff
@@ -0,0 +1,755 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Pavol Ipoth <pavol.ipoth@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import (absolute_import, division, print_function)
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+author:
+    - "Pavol Ipoth <pavol.ipoth@gmail.com>"
+module: lvol_thin
+version_added: 2.4
+short_description: Configure Thin LVM logical volumes
+description:
+  - This module creates, removes or resizes, converts thin logical volumes.
+options:
+  vg:
+    description:
+    - The volume group this logical volume is part of.
+    required: true
+  lv:
+    description:
+    - The name of the logical volume.
+    required: true
+  size:
+    description:
+    - The size of the logical volume, by
+      default in megabytes or optionally with one of [bBsSkKmMgGtTpPeE] units; or
+      as a percentage of [VG|PVS|FREE]; +size is not implemented, as it is by
+      design not indempodent
+      Float values must begin with a digit.
+  state:
+    choices: [ "present", "absent" ]
```

Comment:
You don't have to quote the values in the list. And no spaces needed at the beginning and the end.